src/tasks/tasks.py

```diff
@@ -2224,3 +2224,26 @@ def get_metrics(self, reset=False):
         f1 = (f11 + f12 + f13) / 3
         return {'accuracy': acc, 'f1': f1, 'precision': pcs, 'recall': rcl}
 
+@register_task('winograd-coreference', rel_path = 'winograd-coref')
+class WinogradCoreferenceTask(SpanClassificationTask):
+    def __init__(self, path,  **kw):
+        self._files_by_split = {'train': "train.json", 'val': "val.json",'test': "test_final.json"}
```

Comment:
these should be jsonl

Reply:
After running the preprocessing script, you will get JSON data instead of jsonl. I can change this, but don't think it's too urgent.